### DISCUSSION TITLE
Custom field editor -- rich text edit

### DISCUSSION THREAD

**Discussion by dhait** [aminalaee/sqladmin#490] [Q&A] [answered]
How would I use a rich text editor such as TinyMCE in a text field on an edit form?

**Comment by GarryGon4ar** [comment]
@dhait have u find a way to use TinyMCE rich text field? I would be grateful if u provide example if u did, thanks

**Reply by aminalaee** [reply, marked as answer]
I just realized this section was already in the docs but needed to add it to the cook part: https://github.com/aminalaee/sqladmin/blob/main/docs/cookbook/using_wysiwyg.md
